setup.cfg

```diff
@@ -43,9 +43,17 @@ python_requires = >= 3.7
 [options.extras_require]
 dev =
     setuptools>=3.7.1
+pandas =
+    pandas>=1.3.5
 keepalive =
     keepalive>=0.5
```

Comment:
Didn't you say that we could get rid of the dependency on `keepalive`?

Reply:
ref: #198 
I'd like to remove keepalive from the dependent libraries in another PR, since it's not relevant for typing

Reply:
Sounds sensible!